### DISCUSSION TITLE
Create more than one body to a request

### DISCUSSION THREAD

**Discussion by marcos-venicius** [Kong/insomnia#5619] [Ideas]
I was using the application, so i'd need to add a new body with different values and i was thinking: if we could have multiple bodys?

like: i choose what is the body i want to send.

for example, i have an endpoint to register an user, and i want to create two users.

so, i create a body and send the request, then i create another body and send other request.
in the next time i can choose which body i want to use to send this request.

i think this is a usefull feature

**Comment by jgilbert** [comment]
I agree. I have an API which we use to create Virtual Server instances and having different pre-configured bodies where I could select Windows vs. Ubuntu, encryption vs. non, etc. would be helpful

**Comment by plollaton** [comment]
I would suggest the same feature, it makes the developer's life easier because we can test many request contents at the same endpoint without duplicating them.

**Comment by adiusz** [comment]
any update on that?

**Comment by meprabin000** [comment]
Checking in on this. It would be nice to have a feature to allow multiple payloads saved for a request.

**Comment by erickgcastro** [comment]
up
